app/controllers/application.js

```diff
@@ -41,4 +46,20 @@ export default class ApplicationController extends Controller {
         return (router.currentRouteName !== 'error404' || session.isAuthenticated)
                 && !router.currentRouteName.match(/(signin|signup|setup|reset)/);
     }
+
+    @action
+    openUpdateTab() {
+        const updateWindow = window.open('', '_blank');
+
+        updateWindow.document.write('Loading...');
+
+        const updateUrl = new URL(this.config.get('hostSettings.update.url'));
+        const ghostIdentityUrl = this.ghostPaths.url.api('identities');
+
+        this.ajax.request(ghostIdentityUrl).then((response) => {
+            const token = response && response.identities && response.identities[0] && response.identities[0].token;
```

Comment:
```suggestion
            const token = response?.identities?.[0]?.token;
```

Reply:
I didn't even switch to my favorite feature of JS 😭